commons/src/store/contacts.ts

```diff
@@ -36,21 +36,24 @@ function initializeContacts() {
     subscribe,
     addContacts: async (
       query: ContactsQuery,
-      offset: number,
-      limit: number,
+      params: {
+        offset: number;
+        limit: number;
+        group?: string;
+      },
     ) => {
       const queryKey = JSON.stringify(query);
       if (
         !contactsMap[queryKey] &&
         (query.component_id || query.access_token)
       ) {
-        if (offset === 0) {
+        if (params.offset === 0) {
```

Comment:
If you go the `queryString` route, this would look like `if (queryString.includes("offset=0")) {`